pression/pluginRegistry.cpp

```diff
@@ -59,8 +59,11 @@ class PluginRegistry
             directories.push_back( getcwd( cwd, MAXPATHLEN ));
 
 #ifdef _WIN32
+            //rgh: does not compile, and seems to be handled by code above.
+            /*
             if( GetModuleFileName( 0, cwd, MAXPATHLEN ) > 0 )
                 directories.push_back( pression::getDirname( cwd ));
```

Comment:
59 adds the current working directory, 62 adds the path of the current executable/library/module. It just happens that cwd is reused for 62, which leads to confusion and is not a good coding style, agreed :) But cwd has already the correct size/length for both calls.